src/qwed_new/core/diagnostics.py

```
@@ -520,9 +523,119 @@ def from_legacy_dict(cls, data: Dict[str, Any], engine: str = "unknown") -> "Dia
+                "policy": "mandatory",
+                "verdict_status": result.status.value,
+                "verdict_proof_ref": result.proof_ref,
+            },
+        )
+
+    # Verify the attestation token
+    try:
+        from .attestation import get_attestation_service
+
+        service = get_attestation_service()
+        is_valid, claims, error = service.verify_attestation(
```

Ruff (0.15.21)

[warning] Unpacked variable `claims` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)